internal/cmd/argus/plans/plans.go

```diff
@@ -0,0 +1,144 @@
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+)
+
+const (
+	limitFlag = "limit"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Limit *int64
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "plans",
+		Short: "Lists all Argus service plans",
+		Long:  "Lists all Argus service plans.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`List all Argus service plans`,
+				"$ stackit argus plans"),
+			examples.NewExample(
+				`List all Argus service plans in JSON format`,
+				"$ stackit argus plans --output-format json"),
+			examples.NewExample(
+				`List up to 10 Argus service plans`,
+				"$ stackit argus plans --limit 10"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("get Argus service plans: %w", err)
+			}
+			plans := *resp.Plans
+			if len(plans) == 0 {
+				projectLabel, err := projectname.GetProjectName(ctx, cmd)
+				if err != nil {
+					projectLabel = model.ProjectId
+				}
+				cmd.Printf("No plans found for project %q\n", projectLabel)
+				return nil
+			}
+
+			// Truncate output
+			if model.Limit != nil && len(plans) > int(*model.Limit) {
+				plans = plans[:*model.Limit]
+			}
+
+			return outputResult(cmd, model.OutputFormat, plans)
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Int64(limitFlag, 0, "Maximum number of entries to list")
+}
+
+func parseInput(cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	limit := flags.FlagToInt64Pointer(cmd, limitFlag)
+	if limit != nil && *limit < 1 {
+		return nil, &errors.FlagValidationError{
+			Flag:    limitFlag,
+			Details: "must be greater than 0",
+		}
+	}
+
+	return &inputModel{
+		GlobalFlagModel: globalFlags,
+		Limit:           limit,
+	}, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *argus.APIClient) argus.ApiListPlansRequest {
+	req := apiClient.ListPlans(ctx, model.ProjectId)
+	return req
+}
+
+func outputResult(cmd *cobra.Command, outputFormat string, plans []argus.Plan) error {
+	switch outputFormat {
+	case globalflags.JSONOutputFormat:
+		details, err := json.MarshalIndent(plans, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal Argus plans: %w", err)
+		}
+		cmd.Println(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.SetHeader("PLAN NAME", "ID", "DESCRIPTION")
```

Comment:
Could you flip name and ID here? In the CLI we usually always show the ID first when it exists